source/pages/events.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react'
+import { Segment, Accordion, Icon, Label, Statistic, Card, Button } from 'semantic-ui-react'
+
+import '../styles/events.scss'
+import NewEvent from '../components/newevent.jsx'
+
+import axios from 'axios'
+const moment = require('moment')
+
+class Events extends Component {
+
+    constructor() {
+        super();
+        this.state = {
+            events: [],
+            modal: false,
+        }
+    }
+
+    async componentWillMount() {
+        //const response = await axios.get('http://points-api.illinoiswcs.org/api/events');
+        const response = await axios.get('http://localhost:3000/api/events');
```

Comment:
should be the commented line right?

Reply:
omg yea i always forget to switch it back lmao